history-of-the-quakers-v4/original/05-book-6-chapter-18.adoc

```diff
@@ -286,11 +308,16 @@ offered himself for bail, and took him to his house that night,
 where he was very kindly entertained.
 The next day he was sent to jail with the following mittimus:
 
-Queen`'s County S.S.
+[.embedded-content-document.legal]
```

Comment:
So yeah, this is probably a mess...not sure about this one!! 😃😬

Reply:
me either, let's go with what you did. 😊